config.py.sample

```diff
@@ -37,6 +37,9 @@ MUSICBRAINZ_CLIENT_SECRET = "CLIENT_SECRET"
 LASTFM_API_KEY = "USE_LASTFM_API_KEY"
 
 
+# Max time in seconds after which the playing_now stream will expire.
+PLAYING_NOW_MAX_DURATION = 5 * 60
```

Comment:
I'm happy with 10
@pinkeshbadjatiya Make sure that you clear playing now if a new listen comes in before the playing now key expires (I don't think we have this)

Let's add a new ticket (or leave LB-80 open with a note) for the more advanced aspects (getting song length from listen metadata or musicbrainz)